src/qbittorrent_telegram_notification.sh

```diff
@@ -1,23 +1,40 @@
 #!/bin/bash
 
-# Bot token
-BOT_TOKEN="ADD_YOUR_TELEGRAM_BOT_TOKEN"
+# Source the configuration file for BOT_TOKEN and CHAT_ID
+BASEDIR=$(dirname "$0")
+source "${BASEDIR}/bot_creds.sh"
 
-# Your chat id
-CHAT_ID="ADD_YOUR_TELEGRAM_CHAT_ID"
-
-# Get variables from qbittorrent
+# Extract specific arguments if needed
 TORRENT_NAME="$1"
+TORRENT_SIZE_BYTES="$2"
```

Comment:
Could you please update the README.md screenshot to include the `%Z` parameter?

Reply:
Updated in latest version of PR